build.js

```diff
@@ -429,36 +437,49 @@ const getExtAttr = (node, name) => {
   return node.extAttrs && node.extAttrs.find((i) => i.name === name);
 };
 
-// https://webidl.spec.whatwg.org/#dfn-exposure-set
-const getExposureSet = (node) => {
-  // step 6-8
+// https://webidl.spec.whatwg.org/#Exposed
+const getExposureSet = (node, scopes) => {
+  // step 6-8 of https://webidl.spec.whatwg.org/#dfn-exposure-set
   const attr = getExtAttr(node, 'Exposed');
   if (!attr) {
     throw new Error(
       `Exposed extended attribute not found on ${node.type} ${node.name}`
     );
   }
-  const globals = new Set();
+  const exposure = new Set();
   switch (attr.rhs.type) {
     case 'identifier':
-      globals.add(attr.rhs.value);
+      exposure.add(attr.rhs.value.replace('GlobalScope', ''));
       break;
     case 'identifier-list':
       for (const {value} of attr.rhs.value) {
-        globals.add(value);
+        exposure.add(value.replace('GlobalScope', ''));
+      }
+      break;
+    case '*':
+      for (const value of scopes) {
+        exposure.add(value);
       }
       break;
     /* istanbul ignore next */
     default:
       throw new Error(`Unexpected RHS for Exposed extended attribute`);
   }
 
-  if (globals.has('DedicatedWorker')) {
-    globals.delete('DedicatedWorker');
-    globals.add('Worker');
+  if (exposure.has('DedicatedWorker')) {
```

Comment:
This is a workaround for something. Can you link to https://github.com/foolip/mdn-bcd-collector/pull/811 and explain why this is needed? At first glance it just seems wrong to do this replacement.